js/client/modules/@arangodb/README.md

```diff
@@ -0,0 +1,128 @@
+=== Arango DB test infrasturcture ===
+== Philosopy ==
+All the code structure and the test infrastructure works in a way to
+- have results in all cases
+- must not lock up, and terminate in a reasonable timeframe under all circumstances
+- waste as less host resources as possible -> 
+  - detect lockups and abort tests (C++ / JS infrastructure)
+  - have a well established timeout and deadline mechanism 
+  - have a subprocess monitoring to abort as fast as possible if 
+- collect as much data from the system as possible
+- compress the data to present the user with a most compact result
+- Timeout is for individual integration testsuites. Its 15 minutes by default.
+- Deadline is for the total load of tests. 
+
+== files concerned ==
+
+| Path / File                                                  | Description                                                                                                 |
+| :----------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------- |
+| `testutils/testing.js`                                       | invoked via `unittest.js` handles module structure for `testsuites`.                                        |
+| `testutils/result-processing.js`                             | contains code to run various analyses on results and/or generate result representations for the user/ci     |
+| `testutils/test-utils.js`                                    | infrastructure for tests like filtering, bucketing, iterating                                               |
+| `testutils/process-utils.js`                                 | manage arango instances, start/stop/monitor SUT-processes                                                   |
+| `testutils/result-processing.js`                             | work with the result structures to produce reports, hit lists etc.                                          |
```

Comment:
result-processing.js is already described in line 20. Did you mean to write about a different file her (or there), or are both descriptions about the same file and should therefore be merged?